src/services/util.js

```diff
@@ -109,7 +119,21 @@ function getYearCijfers (data, last = null) {
         color: cijfer.color,
         display: (cijfer.waarde / totalWaarde[cijfer.jaar]) > 0.075 ? displayWaarde(cijfer) : '',
         cijfer
-      }))))
+      })))
+  ).filter(({ x }) => {
+    if (odd) return x % 2 === 0
+    if (even) return x % 2 > 0
+    return true
+  }).filter(({ x }) => {
+    if (before) return x > before
```

Comment:
Looks like the comparison is just the other way around here also.
Before => x < before
Before = 2000, 1999 < 2000 is true
Or have I misunderstood something here?